src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  max-width: 100%;
+  align-items: center;
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.header__images {
+  display: block;
+}
+
+.header__nav {
+  height: 100%;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  align-items: center;
+}
+
+.nav__link {
+  display: flex;
+  justify-content: center;
+  text-decoration: none;
+  margin-left: 20px;
+  font-size: 12px;
+  color: #000;
+  text-transform: uppercase;
+  line-height: 60px;
+  text-align: center;
+  align-items: center;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  margin: 0;
```

Comment:
Please remove margin-left for first nav item, but not for is-active. If, for example, the third element becomes active, then its margin-right will also be deleted

Reply:
I don't know how to do it